infrastructure/src/main/kotlin/no/kartverket/matrikkel/bygning/infrastructure/database/repositories/bygning/BygningRepositoryImpl.kt

```diff
@@ -0,0 +1,99 @@
+package no.kartverket.matrikkel.bygning.infrastructure.database.repositories.bygning
+
+import com.fasterxml.jackson.databind.SerializationFeature
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
+import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+import com.fasterxml.jackson.module.kotlin.readValue
+import no.kartverket.matrikkel.bygning.application.bygning.BygningRepository
+import no.kartverket.matrikkel.bygning.application.models.Bruksenhet
+import no.kartverket.matrikkel.bygning.application.models.ids.BruksenhetBubbleId
+import no.kartverket.matrikkel.bygning.application.models.ids.BruksenhetId
+import no.kartverket.matrikkel.bygning.application.models.ids.BygningId
+import no.kartverket.matrikkel.bygning.infrastructure.database.executeQueryAndMapPreparedStatement
+import no.kartverket.matrikkel.bygning.infrastructure.database.prepareAndExecuteUpdate
+import no.kartverket.matrikkel.bygning.infrastructure.database.withTransaction
+import org.postgresql.util.PGobject
+import java.sql.Timestamp
+import java.util.*
+import javax.sql.DataSource
+
+
+class BygningRepositoryImpl(private val dataSource: DataSource) : BygningRepository {
+    val objectMapper = jacksonObjectMapper()
+        .registerModule(JavaTimeModule())
+        .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
+
+    override fun saveBruksenhet(bruksenhet: Bruksenhet) {
+        val bruksenhetToSave = bruksenhet.toDTO()
```

Comment:
Noe sånt i stedet kanskje? Litt smak og behag men.
```suggestion
        val bruksenhetDTO= bruksenhet.toDTO()
```